src/OpenConext/ProfileBundle/Resources/config/services.yml

```diff
@@ -12,6 +12,16 @@ services:
         tags:
             - { name: kernel.event_subscriber }
 
+    profile.attribute_support.email_from:
+        class: OpenConext\Profile\Value\EmailAddress
```

Comment:
Oh, please mark all services as private, since we never `get()` them from the container.